Framework/Reflectometry/test/SaveReflCustomAsciiTest.h

```diff
@@ -8,20 +8,20 @@
 
 #include "MantidAPI/AlgorithmManager.h"
 #include "MantidAPI/AnalysisDataService.h"
-#include "MantidDataHandling/SaveReflCustomAscii.h"
 #include "MantidDataObjects/Workspace2D.h"
 #include "MantidDataObjects/WorkspaceCreation.h"
 #include "MantidHistogramData/Points.h"
+#include "MantidReflectometry/SaveReflCustomAscii.h"
```

Comment:
You can remove this algorithm altogether, this is deprecated in favour of SaveReflectometryAscii.

Reply:
This is currently still used by the ISIS Reflectometry GUI. We'll look at removing it in #24429